### PR TITLE
Use Travis for continuous intergration tests running

### PR DESCRIPTION
**Pull request by rutsky**
[Travis CI](http://travis-ci.org/) is a continuous integration system that is free of charge (and AFAIK open source), allows to run specified tests in specified set of environments (different Python versions, different libraries versions etc), and has integration with Github.

I prepared configuration file for Travis CI, that installs pygdal and runs `python -c "from osgeo import gdal, ogr, osr"` as test.

Travis runs test in customized Ubuntu 12.04 virtual machines, I managed to install (with some pain) into VM GDAL 1.11.2 from Ubuntu 15.04.

Currently only Python 2.7 and 3.4 are enabled, other versions may be enabled if needed.

Tutorial on how to use Travis can be found here: http://docs.travis-ci.com/user/getting-started/
Long story short, to enable Travis on your account you need to:

1. Login on https://travis-ci.org with your Github account.
2. In your Travis profile settings https://travis-ci.org/profile activate Travis for pygdal repository.

Then, for all pull requests and pushes Travis will fetch code and run tests on them.

Here how it looks inside my fork of pygdal: https://travis-ci.org/rutsky/pygdal
You can find there logs for each build.

Main goal of this work is to provide you and other developers with some environment, where pygdal can be tested.

### CODE REVIEW

**Comment by dezhin** [comment]
Wow! It need some time. I'll try this and fix #8 on next week.

**Comment by dezhin** [comment]
Thank you for your work. Everything seems to be working.
https://travis-ci.org/dezhin/pygdal
